includes/settings/settings-page.php

```diff
@@ -346,6 +350,10 @@ public function display_settings_page() {
 							$section['callback']();
 						}
 
+						if ( 'experiments' === $section_id ) {
+							$this->render_ongoing_experiments_section();
+						}
+
```

Comment:
Isn't it a generic page that should get its content from a dedicated file? (a file that is dedicated to the content of the experiment?).

Reply:
This is how WordPress works, I don't think we can add setting fields in a different way.
@kobizz What do you think?